02Lesson/02Project/css/style.css

```diff
@@ -1,24 +1,59 @@
+
 body {
-  width: 75%;
-  margin: 0 auto;
+  background-image:url("http://www.randco.com/media/wysiwyg/RCo-PDP-Background-Images/outerspace.jpg");

```

Comment:
Okay... so in this case there should be a space between background-image: and url but the width and margin are correctly formatted?